core/sql/executor/ExStats.cpp

```diff
@@ -2357,11 +2357,19 @@ Int64 ExHdfsScanStats::getNumVal(Int32 i) const
 
 NABoolean ExHdfsScanStats::filterForSEstats(struct timespec currTimespec, Lng32 filter)
```

Comment:
I wonder if the type of filter argument should be Int64 here, since Int64 / 1000000LL could be a value larger than max(Int32).